integration-tests/hibernate-reactive-panache/src/test/java/io/quarkus/it/panache/reactive/TestReactiveTransactionTest.java

```diff
@@ -6,17 +6,20 @@
 import io.quarkus.hibernate.reactive.panache.Panache;
 import io.quarkus.test.TestReactiveTransaction;
 import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.vertx.RunOnVertxContext;
 import io.quarkus.test.vertx.UniAsserter;
 
 @QuarkusTest
 public class TestReactiveTransactionTest {
 
+    @RunOnVertxContext
     @TestReactiveTransaction
```

Comment:
Can we make that one imply the previous one?

Reply:
Hmm, `@RunOnVertxContext` is not an interceptor binding so unfortunately we can't leverage the fact that interceptor bindings are transitive. And the `io.quarkus.test.vertx.RunOnVertxContextTestMethodInvoker` is using reflection to detect this annotation so we can't transform the annotations either.

But it seems that `RunOnVertxContextTestMethodInvoker` is already used for methods annotated with `TestReactiveTransaction` so the only thing we're missing is the [vertx duplicated context](https://github.com/quarkusio/quarkus/blob/2.15.0.Final/test-framework/vertx/src/main/java/io/quarkus/test/vertx/RunOnVertxContextTestMethodInvoker.java#L99-L109). 

@geoand Do you think it would make sense to return `true` on [this line](https://github.com/quarkusio/quarkus/blob/2.15.0.Final/test-framework/vertx/src/main/java/io/quarkus/test/vertx/RunOnVertxContextTestMethodInvoker.java#LL105C11-L105C11)? I.e. use the duplicated context by default?

Reply:
Actually, we should only return `true` if `@RunOnVertxContext` is not present, but `@TestReactiveTransaction` is...

Reply:
I think that makes sense

Reply:
Ok, I'll try to update the code and we'll see if the change makes it in ;-)